gravitee-gateway-services/gravitee-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/apikeys/ApiKeysCacheService.java

```diff
@@ -219,4 +220,12 @@ private void stopRefresher(Api api) {
             }
         }
     }
+
+    public static String buildCacheKey(ApiKey apiKey) {
+        return buildCacheKey(apiKey.getApi(), apiKey.getKey());
+    }
+
+    public static String buildCacheKey(String api, String key) {
+        return String.format("%s.%s", api, key);
+    }
```

Comment:
Why not creating a new object with two constructors and with a getter ?
Or creating an ApiKeyCache which holds the Map, and manage directly this key formatting ?
I am not a big fan of that static method inside ApiKeysCacheService.
wdyt ?

Reply:
yes, i refactored it, and let you check